### PR TITLE
Add mirage-block-xen.1.5.1

### PR DESCRIPTION
**Pull request by djs55**
Signed-off-by: David Scott <dave@recoil.org>

### CODE REVIEW

File: packages/mirage-block-xen/mirage-block-xen.1.5.1/opam [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+]
+
+depends: [
+  "ocamlfind" {build}
+  "jbuilder"  {build & >="1.0+beta9"}
+  "cmdliner"
+  "logs"
+  "stringext"
+  "lwt" {>= "2.4.3"}
+  "cstruct" {>= "1.9.0"}
```
Comment (avsm) [inline packages/mirage-block-xen/mirage-block-xen.1.5.1/opam]: does this require `cstruct-lwt` as well?

Comment (djs55) [inline packages/mirage-block-xen/mirage-block-xen.1.5.1/opam]: No sign of any uses of `Lwt_cstruct` (unlike `mirage-block-unix`)

Comment (avsm) [inline packages/mirage-block-xen/mirage-block-xen.1.5.1/opam]: ah yes because not Unix...

**Comment by djs55** [comment]
It looks like the `mirage-xen-minios 0.8.0` package fails to build on alpine.

**Comment by avsm** [comment]
known breakage for now, not a blocker to merge

**Pull request merged**